variants/FK407M1/variant.h

```diff
@@ -0,0 +1,210 @@
+ *        Headers
+ *----------------------------------------------------------------------------*/
+#include "PeripheralPins.h"
+
+#ifdef __cplusplus
+extern "C"{
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+extern const PinName digitalPin[];
+// Left Side
+#define PD10  0
+#define PD11  1
+#define PB15  2
+#define PD9   3
+#define PB12  4
+#define PD8   5
+#define PB14  6
+#define PB11  7
+#define PE8   8
+#define PB13  9
+#define PB10 10
+#define PE7  11
+#define PE14 12
+#define PE15 13
+#define PE12 14
+#define PE13 15
+#define PE11 16
+#define PE10 17
+#define PE9  18
+#define PB1  19
+#define PB2  20
+#define PC5  21
+#define PB0  22
+#define PA7  23
+#define PC4  24
+#define PA6  25
+#define PA5  26
+#define PA3  27
+#define PA4  28
+#define PC3  29
+#define PA0  30
+#define PA2  31
+#define PA1  32
+#define PD13 33
+#define PD14 34
+#define PC0  35
+#define PC1  36
+#define PC2  37
+// Right Side
+#define PA12 38
+#define PA11 39
+#define PA10 40
+#define PD12 41
+#define PC9  42
+#define PA9  43
+#define PA8  44
+#define PC10 45
+#define PC11 46
+#define PC12 47
+#define PD0  48
+#define PD1  49
+#define PD2  50
+#define PD3  51
+#define PD4  52
+#define PD5  53
+#define PB5  54
+#define PB4  55
+#define PB7  56
+#define PB6  57
+#define PB9  58
+#define PB8  59
+#define PE1  60
+#define PE0  61
+#define PE2  62
+#define PE6  63
+#define PE3  64
+#define PE5  65
+#define PB3  66
+#define PE4  67
+#define PD6  68
+#define PD7  69
+#define PC7  70
+#define PC8  71
+#define PD15 72
+#define PC6  73
+#define PC13 74
+// This must be a literal
+#define NUM_DIGITAL_PINS        91
+// This must be a literal with a value less than or equal to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       16
+#define NUM_ANALOG_FIRST        75
+
+// Below ADC, DAC and PWM definitions already done in the core
+// Could be redefined here if needed
+// ADC resolution is 12bits
+//#define ADC_RESOLUTION          12
+//#define DACC_RESOLUTION         12
+
+// PWM resolution
+//#define PWM_RESOLUTION          8
+//#define PWM_FREQUENCY           1000
+//#define PWM_MAX_DUTY_CYCLE      255
+
+// On-board LED pin number
+#define LED_D1                  PC13  //TBD
+#define LED_BUILTIN             LED_D1
+
+// On-board user button
+#define BTN_KEY                 PA15  // TBD
+#define USER_BTN                BTN_KEY
+
+// Below SPI and I2C definitions already done in the core
+// Could be redefined here if differs from the default one
+// SPI Definitions
+
+#define PIN_SPI_MOSI            PB5 // NRF24 connector & W25Q16 (on board flash)
+#define PIN_SPI_MISO            PB4 // NRF24 connector & W25Q16 (on board flash)
+#define PIN_SPI_SCK             PB3 // NRF24 connector & W25Q16 (on board flash)
+
+// I2C Definitions
+#define PIN_WIRE_SDA            PB9
```

Comment:
You set the default pins for Wire instance to be PB9 and PB8. If you think it should be PB7/PB6 then update it here.

Reply:
Ah lol OK - sleep deprived, copy and paste from Black407XX

Not sure why that variant chose to go with the alternate I2C1 pins.

Reply:
difficult to tell maybe for PCB reasons...

Reply:
I've eyeballed the board and there's no reason why it should not be on the default pins.   Odd one.

Reply:
Just checking layout and schematics, I also see no reason. So, up to you to choose the default I2C pins used by Wire instance.